FieldMetaBuilderSpike.ipynb

```diff
@@ -65,23 +65,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "query\n",
-      "   region{'id': '01'}\n",
-      "      name\n",
-      "      AENW01\n",
-      "         year\n",
-      "      AENW06\n",
-      "         year\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
this and upcoming few files are just cleaning notebooks, I like to keep them without outputs. this shall go soon in the pre commit hooks